packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -2064,13 +2066,20 @@ function updateSuspensePrimaryChildren(
   if (currentFallbackChildFragment !== null) {
     // Delete the fallback child fragment
     currentFallbackChildFragment.nextEffect = null;
-    currentFallbackChildFragment.effectTag = Deletion;
+    currentFallbackChildFragment.effectTag =
+      (currentFallbackChildFragment.effectTag & StaticMask) | Deletion;
     workInProgress.firstEffect = workInProgress.lastEffect = currentFallbackChildFragment;
     const deletions = workInProgress.deletions;
     if (deletions === null) {
       workInProgress.deletions = [currentFallbackChildFragment];
       // TODO (effects) Rename this to better reflect its new usage (e.g. ChildDeletions)
       workInProgress.effectTag |= Deletion;
+
+      // We are deleting a subtree that may contain a passive effect.
+      // Mark the parent so we traverse this path after commit and run any unmount functions.
+      // This may cause us to traverse unnecessarily in some cases, but effects are common,
+      // and the cost of over traversing is small (just the path to the deleted node).
+      workInProgress.subtreeTag |= PassiveSubtreeTag;
```

Comment:
Same comment as https://github.com/facebook/react/pull/19374/files#r462472081

Reply:
Right. I assumed that comment applied to all 4 places we were bubbling :)